src/main/java/org/junitpioneer/jupiter/ExpectedToFailExtension.java

```diff
@@ -31,6 +32,11 @@ public void interceptTestMethod(Invocation<Void> invocation, ReflectiveInvocatio
 
 	private static void invokeAndInvertResult(Invocation<Void> invocation, ExtensionContext extensionContext)
 			throws Throwable {
+		ExpectedToFail expectedToFail = getExpectedToFailAnnotation(extensionContext);
+		if (expectedToFail.withExceptions().length == 0) {
+			fail("@ExpectedToFail withExceptions must not be empty");
```

Comment:
This is not an `AssertionFailedError`, since it does not originate from the test itself - should be an `ExtensionConfigurationException`.